enhancements/console/workflow-guides.md

```diff
@@ -0,0 +1,175 @@
+   ![operators](https://raw.githubusercontent.com/jhadvig/images/master/operators.png)
+   * ...TBD
+
+#### Resource Checker Extension
+
+In order for the user to see what resources need to be created in terms of a specific Workflow Guide, we will display list of those resources in a right sidebar similar to the proposed [Solution Explorer](https://marvelapp.com/236ge4ig/screen/69628466). Each of the resource items will be prefixed with an icon which will indicate if the resource exists or not.
+
+For the Resource Check we will need to define a new CRD that will contain list of resources. For each resource, following fields will need to be defined:
+   * Kind - string - Kind of the resource.
+   * Name - string - Name of the resource.
+   * Namespace - string - Namespace of the resource. (Optional) 
+   * CurrentNamespace - bool - Indicates whether the resource should be check in user's current namespace. If set will be honored instead of the `Namespace` field. (Optional) 
+   * ConsoleYAMLSampleName - string - Reference to a ConsoleYAMLSample CR that contains the definition of the resource. If defined the resource item will become a link which would navigate user to Console's YAML editor which would be prefilled with referenced YAML sample. (Optional) 
+
+ResourceCheck spec:
+```
+apiVersion: console.openshift.io/v1
+kind: ResourceCheck
+metadata:
+  name: logging-operator
+spec:
+  resources:
+  - kind: Namespace
+    apiVersion: v1
+    name: openshift-logging
+  - kind: ClusterLogging
+    apiVersion: logging.openshift.io/v1
+    name: openshift-logging
+  - ...
+```
+
+The check itself would be done by a GET API call from the OpenShift Console. The calls could be done:
+   * Serial - Makes API  call for one resource at a time (top to bottom).
+   * Parallel - Makes API calls for all the resource at a time.
+
+The Resource Checker sidebar will also contain link to the appropriate Workflow Guide, so user can go though extensive guide on the GitHub page and also have his progress check on the Resource Check sidebar.
+
+TBD: 
+  * Figure out potentional security issues, where user could define checks of resources in different namespaces then he is authorized.
+  * To determin name of some resoruces could be tricky, since some resource names are generated by pattern, e.g. ClusterServiceVersion: `<resource name>.<resource version>` - `clusterlogging.4.4.0-202005270305`.
+
+#### Air Gapped Environments
+
+If we decide to suppport this case we should have a `workflow-guides-operator`. It's image will contain all the official Workflow Guides. Upon `workflow-guides-operator` image build `openshift/workflow-guides` repository will be cloned into the image so it contains all the official Workflow Guides. The `workflow-guides-operator` will run a pod with a small server that would handle serving of the Workflow Guides.
```

Comment:
I don't know that we need to worry about the browser being disconnected in these environments. As I understand, it's requests that come from inside the cluster to the outside that we need to worry about. Browser links to GitHub are probably OK.

Reply:
oh, then I guess this is not an issue then